aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -263,8 +299,23 @@ Tensor& addmm_out_cuda_impl(Tensor& result, const Tensor& self, const Tensor& ma
               result_ptr,
               result_ld);
         });
+    switch (activation) {
+      case Activation::RELU:
+        at::relu_(const_cast<Tensor&>(*result_));
+        break;
+      case Activation::GELU:
+        result_ = c10::MaybeOwned<Tensor>::owned(at::gelu(*result_));
+        break;
+      default: break;
+    }
   }
 
+#if defined(CUDA_VERSION) && CUDA_VERSION >= 11000 && CUDA_VERSION < 11060 && !defined(_MSC_VER)
+  if (useLtInterface && activation == Activation::GELU) {
```

Comment:
can you also leave a comment here explaining why gelu has to be done separately

Reply:
OK I'll make the above gelu comment into a NOTE and reference it